src/lib/config.js

```diff
@@ -81,6 +81,17 @@ var getMongoDbPort = function() {
   return mongoPort;
 };
 
+/**
+ *  @returns boolean to define the RS as a configsvr or not. Default is false
+ */
+var isConfigRS = function() {
+  var configsvr = process.env.CONFIG_SVR || false;
+  if (configsvr) {
```

Comment:
Fair point indeed. I'm not too much of a fan to add a new dependency. What about a regex validation, would that be fine for you?
Like so:
```
var configsvr = /^(?:y|yes|true|1)$/i.test((process.env.CONFIG_SVR || '').trim())
```